### DISCUSSION TITLE
AudioDeviceModule

### DISCUSSION THREAD

**Discussion by hussainasad6696** [devopvoid/webrtc-java#63] [Q&A]
Hi, my video communication is successful but for audio as per the wrapper i used AudioDeviceModule() but it started to crash the application with UnsatisfiedLinkError but then i did mvn install but that started to give error
#
# Fatal error in: ../../rtc_base/win/scoped_com_initializer.cc, line 41
# last system error: 1150
# Check failed: ((HRESULT)0x80010106L) != hr_ (-2147417850 vs. -2147417850)
# Invalid COM thread model change (MTA->STA)#
kindly help.

**Comment by devopvoid** [comment]
Fixed with issue #181.
